### ISSUE TITLE
Can we use the same column name in associated tables?

### ISSUE THREAD

**Issue by jmatsu** [gfx/Android-Orma#216]
When a table has a relation to another table, is there any way to handle the same column name in them?

Below example gives us an error which says `ambiguous column name: name`.

```
@Table
class ModelA {
  @Column
  public ModelB b;

  @Column
  public String name;

  @PrimaryKey(auto = false)
  public long a_id = 0L
}

@Table
class ModelB {
  @Column
  public String name;

  @PrimaryKey(auto = false)
  public long b_id = 0L
}
```

Built select query (by `OrmaDatabase.builder(this).build().selectFromModelA().toList()` ) is

```sql
SELECT `a_id, `name`, `name`, `b_id` FROM `ModelA` JOIN `ModelB` ON `ModelA`.`b` = `ModelB`.`b_id` LIMIT 0,2000
```

**Comment by gfx** [comment]
I guess the latest version has no suck problem. Can you check the version of Orma?

**Comment by jmatsu** [comment]
I am using `2.3.0`. It is latest, right? I got this problem with this version.
My build.gradle - [here](https://github.com/jmatsu/MyPlayground/blob/play/orma/app/build.gradle)

**Comment by gfx** [comment]
Looks good. Will look into it.

**Comment by gfx** [comment]
Released v2.3.1. Can you try it please?

**Comment by jmatsu** [comment]
I confirmed that v2.3.1 works great! Thank you so much!
